tests/plugins/link/link.js

```diff
@@ -562,6 +573,43 @@
 					dialog.hide();
 				} );
 			} );
+		},
+
+		// (#2154)
+		'test telephone number link': function() {
+			var editor = this.editor,
+				bot = this.editorBot;
+
+			bot.setHtmlWithSelection( '^' );
+			bot.dialog( 'link', function( dialog ) {
+				var expectedNumber = '123456789',
+					input,
+					stub = this._stubs.alert = sinon.stub( window, 'alert' );
```

Comment:
TBH creating `this._stubs` and whole `tearDown` logic seems a little bit like overkill. We got only this one stub, which could be restored with one line of code instead of current 7. Keeping whole stubbing code inside test itself also improves readability.